marimo/_plugins/stateless/conftest.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2023 Marimo. All rights reserved.
```

Comment:
this is copy-paste from another conftest. is there a better way to share this?

Reply:
For now, not really. I need to refactor our tests to live in their own directory, then we'll just put the `conftest.py` at the top level of `_tests`. 

In the meantime can you move this file to `marimo/_plugins/`?

Reply:
sounds good - we can refactor in a followup. im pretty sure you already have a ticket for this